bitnami/spark/values-production.yaml

```diff
@@ -243,18 +243,18 @@ security:
   ## RPC configuration
   ##
   rpc:
-    authenticationEnabled: "yes"
-    encryptionEnabled: "yes"
+    authenticationEnabled: true
+    encryptionEnabled: true
```

Comment:
These changes probably require a bump in the  major version. wdyt @javsalgar @juan131